homeassistant/components/tts/__init__.py

```diff
@@ -273,9 +273,7 @@ def async_register_engine(self, engine, provider, config):
             provider.name = engine
         self.providers[engine] = provider
 
-    async def async_get_url(
-        self, engine, message, cache=None, language=None, options=None
-    ):
+    async def get_url(self, engine, message, cache=None, language=None, options=None):
```

Comment:
This seems to have sneaked through the S/R.

Reply:
😅 I checked each before staging it, afterwards resettled I staged and run the full suite... missed it 😞 thanks! Will fix that